<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="5" total="16" passed="11">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="18126" started-at="2017-02-23T20:16:32Z" finished-at="2017-02-23T20:16:50Z">
    <groups>
      <group name="ilp_ledger_adapater_prepare_fulfilled_transfer">
        <method signature="ILPLedgerAdapterFunctionalTest.getFulfilledTransfer_ForFulfillmentThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getFulfilledTransfer_ForFulfillmentThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getFulfilledTransfer_ForValidFulfilledTransfer_ShouldReturn200_ShouldReturnTransferDetails(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getFulfilledTransfer_ForValidFulfilledTransfer_ShouldReturn200_ShouldReturnTransferDetails" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
      </group> <!-- ilp_ledger_adapater_prepare_fulfilled_transfer -->
      <group name="ilp_ledger_adapater_prepare_fulfill_transfer">
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForUnmetCondition_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForUnmetCondition_ShouldReturn422_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForValidRequest_ShouldReturn200_ShouldReturnValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForValidRequest_ShouldReturn200_ShouldReturnValidResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
      </group> <!-- ilp_ledger_adapater_prepare_fulfill_transfer -->
      <group name="ilp_ledger_adapater_prepare_transfer">
        <method signature="ILPLedgerAdapterFunctionalTest.getFulfilledTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getFulfilledTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getPreparedTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getPreparedTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getPreparedTransfer_ForTransferThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getPreparedTransfer_ForTransferThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getPreparedTransfer_ForValidTransfer_ShouldReturn200_ShouldReturnValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getPreparedTransfer_ForValidTransfer_ShouldReturn200_ShouldReturnValidResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForEntityThatAlreadyExists_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForEntityThatAlreadyExists_ShouldReturn422_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForValidSenderAndReceiver_ShouldReturn200_ShouldReturnValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForValidSenderAndReceiver_ShouldReturn200_ShouldReturnValidResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
      </group> <!-- ilp_ledger_adapater_prepare_transfer -->
      <group name="ilp_ledger_adapater_all">
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForUnmetCondition_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForUnmetCondition_ShouldReturn422_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.fulfillTransfer_ForValidRequest_ShouldReturn200_ShouldReturnValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForValidRequest_ShouldReturn200_ShouldReturnValidResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getFulfilledTransfer_ForFulfillmentThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getFulfilledTransfer_ForFulfillmentThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getFulfilledTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getFulfilledTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getFulfilledTransfer_ForValidFulfilledTransfer_ShouldReturn200_ShouldReturnTransferDetails(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getFulfilledTransfer_ForValidFulfilledTransfer_ShouldReturn200_ShouldReturnTransferDetails" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getPreparedTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getPreparedTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getPreparedTransfer_ForTransferThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getPreparedTransfer_ForTransferThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.getPreparedTransfer_ForValidTransfer_ShouldReturn200_ShouldReturnValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getPreparedTransfer_ForValidTransfer_ShouldReturn200_ShouldReturnValidResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForEntityThatAlreadyExists_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForEntityThatAlreadyExists_ShouldReturn422_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
        <method signature="ILPLedgerAdapterFunctionalTest.prepareTransfer_ForValidSenderAndReceiver_ShouldReturn200_ShouldReturnValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForValidSenderAndReceiver_ShouldReturn200_ShouldReturnValidResponse" class="com.l1.interop.ILPLedgerAdapterFunctionalTest"/>
      </group> <!-- ilp_ledger_adapater_all -->
    </groups>
    <test name="Default test" duration-ms="18126" started-at="2017-02-23T20:16:32Z" finished-at="2017-02-23T20:16:50Z">
      <class name="com.l1.interop.ILPLedgerAdapterFunctionalTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="setup" is-config="true" duration-ms="867" started-at="2017-02-23T13:16:32Z" finished-at="2017-02-23T13:16:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="beforeClass" is-config="true" duration-ms="1" started-at="2017-02-23T13:16:33Z" finished-at="2017-02-23T13:16:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="fulfillTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse" duration-ms="6" started-at="2017-02-23T13:16:33Z" data-provider="fulfill_transfer_negative" finished-at="2017-02-23T13:16:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fulfillTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse -->
        <test-method status="PASS" signature="fulfillTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse" duration-ms="1" started-at="2017-02-23T13:16:33Z" data-provider="fulfill_transfer_negative" finished-at="2017-02-23T13:16:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fulfillTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse -->
        <test-method status="PASS" signature="fulfillTransfer_ForUnmetCondition_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForUnmetCondition_ShouldReturn422_ShouldReturnErrorResponse" duration-ms="1" started-at="2017-02-23T13:16:33Z" data-provider="fulfill_transfer_negative" finished-at="2017-02-23T13:16:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fulfillTransfer_ForUnmetCondition_ShouldReturn422_ShouldReturnErrorResponse -->
        <test-method status="PASS" signature="fulfillTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse" duration-ms="1" started-at="2017-02-23T13:16:33Z" data-provider="fulfill_transfer_negative" finished-at="2017-02-23T13:16:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fulfillTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse -->
        <test-method status="PASS" signature="fulfillTransfer_ForValidRequest_ShouldReturn200_ShouldReturnValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="fulfillTransfer_ForValidRequest_ShouldReturn200_ShouldReturnValidResponse" duration-ms="1" started-at="2017-02-23T13:16:33Z" data-provider="fulfill_transfer_positive" finished-at="2017-02-23T13:16:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[bob]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[alice]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fulfillTransfer_ForValidRequest_ShouldReturn200_ShouldReturnValidResponse -->
        <test-method status="PASS" signature="getFulfilledTransfer_ForFulfillmentThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getFulfilledTransfer_ForFulfillmentThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse" duration-ms="1" started-at="2017-02-23T13:16:33Z" data-provider="get_fulfilled_transfer_negative" finished-at="2017-02-23T13:16:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFulfilledTransfer_ForFulfillmentThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse -->
        <test-method status="PASS" signature="getFulfilledTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getFulfilledTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse" duration-ms="1" started-at="2017-02-23T13:16:33Z" data-provider="get_prepared_transfer_negative" finished-at="2017-02-23T13:16:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFulfilledTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse -->
        <test-method status="PASS" signature="getFulfilledTransfer_ForValidFulfilledTransfer_ShouldReturn200_ShouldReturnTransferDetails(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getFulfilledTransfer_ForValidFulfilledTransfer_ShouldReturn200_ShouldReturnTransferDetails" duration-ms="1" started-at="2017-02-23T13:16:33Z" data-provider="get_fulfilled_transfer_positive" finished-at="2017-02-23T13:16:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[bob]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[alice]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getFulfilledTransfer_ForValidFulfilledTransfer_ShouldReturn200_ShouldReturnTransferDetails -->
        <test-method status="PASS" signature="getPreparedTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getPreparedTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse" duration-ms="1" started-at="2017-02-23T13:16:33Z" data-provider="get_prepared_transfer_negative" finished-at="2017-02-23T13:16:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPreparedTransfer_ForInvalidURIParameterInRequest_ShouldReturn400_ShouldReturnErrorResponse -->
        <test-method status="PASS" signature="getPreparedTransfer_ForTransferThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getPreparedTransfer_ForTransferThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse" duration-ms="14729" started-at="2017-02-23T13:16:33Z" data-provider="get_prepared_transfer_negative" finished-at="2017-02-23T13:16:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPreparedTransfer_ForTransferThatDoesNotExist_ShouldReturn404_ShouldReturnErrorResponse -->
        <test-method status="FAIL" signature="getPreparedTransfer_ForValidTransfer_ShouldReturn200_ShouldReturnValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="getPreparedTransfer_ForValidTransfer_ShouldReturn200_ShouldReturnValidResponse" duration-ms="659" started-at="2017-02-23T13:16:48Z" data-provider="get_prepared_transfer_positive" finished-at="2017-02-23T13:16:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[bob]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[alice]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <exception class="io.restassured.path.json.exception.JsonPathException">
            <message>
              <![CDATA[Failed to parse the JSON document]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:930)
	at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:895)
	at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:975)
	at io.restassured.path.json.JsonPath.get(JsonPath.java:201)
	at io.restassured.path.json.JsonPath.getString(JsonPath.java:351)
	at com.l1.interop.ILPLedgerAdapterFunctionalTest.getPreparedTransfer_ForValidTransfer_ShouldReturn200_ShouldReturnValidResponse(ILPLedgerAdapterFunctionalTest.java:470)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading 'F', no possible valid JSON value or punctuation could be recognized.
	at groovy.json.JsonLexer.nextToken(JsonLexer.java:85)
	at groovy.json.JsonLexer$nextToken.call(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
	at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:893)
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:928)
	... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- io.restassured.path.json.exception.JsonPathException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getPreparedTransfer_ForValidTransfer_ShouldReturn200_ShouldReturnValidResponse -->
        <test-method status="FAIL" signature="prepareTransfer_ForEntityThatAlreadyExists_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForEntityThatAlreadyExists_ShouldReturn422_ShouldReturnErrorResponse" duration-ms="599" started-at="2017-02-23T13:16:49Z" data-provider="prepare_transfer_positive" finished-at="2017-02-23T13:16:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[bob]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[alice]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <exception class="io.restassured.path.json.exception.JsonPathException">
            <message>
              <![CDATA[Failed to parse the JSON document]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:930)
	at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:895)
	at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:975)
	at io.restassured.path.json.JsonPath.get(JsonPath.java:201)
	at io.restassured.path.json.JsonPath.getString(JsonPath.java:351)
	at com.l1.interop.ILPLedgerAdapterFunctionalTest.prepareTransfer_ForEntityThatAlreadyExists_ShouldReturn422_ShouldReturnErrorResponse(ILPLedgerAdapterFunctionalTest.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading 'F', no possible valid JSON value or punctuation could be recognized.
	at groovy.json.JsonLexer.nextToken(JsonLexer.java:85)
	at groovy.json.JsonLexer$nextToken.call(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
	at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:893)
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:928)
	... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- io.restassured.path.json.exception.JsonPathException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTransfer_ForEntityThatAlreadyExists_ShouldReturn422_ShouldReturnErrorResponse -->
        <test-method status="FAIL" signature="prepareTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse" duration-ms="380" started-at="2017-02-23T13:16:49Z" data-provider="prepare_transfer_negative" finished-at="2017-02-23T13:16:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
JSON path error_id doesn't match.
Expected: InvalidBodyError
  Actual: null
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path error_id doesn't match.
Expected: InvalidBodyError
  Actual: null

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:80)
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:74)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:470)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:632)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at io.restassured.internal.ResponseSpecificationImpl.content(ResponseSpecificationImpl.groovy:264)
	at io.restassured.specification.ResponseSpecification$content$3.callCurrent(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:190)
	at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:247)
	at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:268)
	at com.l1.interop.ILPLedgerAdapterFunctionalTest.prepareTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse(ILPLedgerAdapterFunctionalTest.java:414)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTransfer_ForInvalidRequestBody_ShouldReturn400_ShouldReturnErrorResponse -->
        <test-method status="FAIL" signature="prepareTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse" duration-ms="251" started-at="2017-02-23T13:16:50Z" data-provider="prepare_transfer_negative" finished-at="2017-02-23T13:16:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected status code <422> doesn't match actual status code <200>.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <422> doesn't match actual status code <200>.

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:80)
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:74)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:470)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:632)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:121)
	at io.restassured.specification.ResponseSpecification$statusCode$2.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:129)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)
	at com.l1.interop.ILPLedgerAdapterFunctionalTest.prepareTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse(ILPLedgerAdapterFunctionalTest.java:375)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTransfer_ForInvalidURIParameter_ShouldReturn400_ShouldReturnErrorResponse -->
        <test-method status="PASS" signature="prepareTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse" duration-ms="0" started-at="2017-02-23T13:16:50Z" data-provider="prepare_transfer_negative" finished-at="2017-02-23T13:16:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[john]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jane]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTransfer_ForUnproceesableEntity_ShouldReturn422_ShouldReturnErrorResponse -->
        <test-method status="FAIL" signature="prepareTransfer_ForValidSenderAndReceiver_ShouldReturn200_ShouldReturnValidResponse(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="prepareTransfer_ForValidSenderAndReceiver_ShouldReturn200_ShouldReturnValidResponse" duration-ms="568" started-at="2017-02-23T13:16:50Z" data-provider="prepare_transfer_positive" finished-at="2017-02-23T13:16:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[bob]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[alice]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[100.00]]>
              </value>
            </param>
          </params>
          <exception class="io.restassured.path.json.exception.JsonPathException">
            <message>
              <![CDATA[Failed to parse the JSON document]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:930)
	at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:895)
	at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:975)
	at io.restassured.path.json.JsonPath.get(JsonPath.java:201)
	at io.restassured.path.json.JsonPath.getString(JsonPath.java:351)
	at com.l1.interop.ILPLedgerAdapterFunctionalTest.prepareTransfer_ForValidSenderAndReceiver_ShouldReturn200_ShouldReturnValidResponse(ILPLedgerAdapterFunctionalTest.java:206)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading 'F', no possible valid JSON value or punctuation could be recognized.
	at groovy.json.JsonLexer.nextToken(JsonLexer.java:85)
	at groovy.json.JsonLexer$nextToken.call(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
	at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:893)
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:928)
	... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- io.restassured.path.json.exception.JsonPathException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTransfer_ForValidSenderAndReceiver_ShouldReturn200_ShouldReturnValidResponse -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.l1.interop.ILPLedgerAdapterFunctionalTest@3972a855]" name="afterClass" is-config="true" duration-ms="0" started-at="2017-02-23T13:16:50Z" finished-at="2017-02-23T13:16:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- com.l1.interop.ILPLedgerAdapterFunctionalTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
